recipes-kernel/linux/linux-nezha-dev.bb

```diff
@@ -14,16 +14,19 @@ KMETA = "kernel-meta"
 # directories that may be missing"
 SRC_URI = " \
         git://github.com/${FORK}/linux.git;name=machine;protocol=https;branch=${BRANCH} \
-        git://git.yoctoproject.org/yocto-kernel-cache;type=kmeta;name=meta;branch=yocto-5.15;destsuffix=${KMETA} \
-        file://0001-riscv-fix-build-with-binutils-2.38.patch \
+        git://git.yoctoproject.org/yocto-kernel-cache;type=kmeta;name=meta;branch=master;destsuffix=${KMETA} \
```

Comment:
We should use `BRANCH` here

Reply:
Do you mean that I shall use a new variable for the KMETA branch here? Maybe nobranch=1 might also be a solution.

Reply:
I think using revelant branch of yocto-kernel-cache is good to have see https://git.yoctoproject.org/yocto-kernel-cache/
but I think since this is a dev recipe I expect it to be latest kernel and hence using master might not be a bad idea.

Reply:
OK, I am fine with either way, just need a decision what to use :)